docs/Installing.asciidoc

```diff
@@ -724,11 +724,11 @@ audit_enabled = 0
 --------------------------------------------------------------------------------
 
 The `audit` section of `/etc/openqa/openqa.ini` allows to exclude some events from logging using
-a space separated blacklist:
+a space separated blocklist:
 [source,ini]
 --------------------------------------------------------------------------------
 [audit]
-blacklist = job_grab job_done
+blocklist = job_grab job_done
```

Comment:
hmm, do you have a migration for this?

Reply:
good point. How about a sed-call in the spec file on upgrade? Or should we handle that in the code itself to allow both values?

Reply:
Falling back to the old value seems like a good approach (I'm not a fan of modifying configuration during the upgrade process).

Reply:
done, still reading both keys